### PR TITLE
security/tor: fix help string, add memory limit; closes #384

### LINKED ISSUE #384: security/tor: wrong description for the "Bridge" option
The help of the "Bridge" option (in the Relaying tab) says: "A bridge is a private relay. It is not visible in the public directory. Check this if you want to become a relay (public)." It should be: "*don't* check this if you want to become a relay", or "check this if you want to become *a bridge*".

### PR DESCRIPTION
**Pull request by fabianfrz**
@paride this should fix both of your issues.
If you have time you can apply this patch using the opnsense-patch utility.
The commit id is 774b2bc

### CODE REVIEW

File: security/tor/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=          tor
-PLUGIN_VERSION=       1.2
+PLUGIN_VERSION=       1.3
```
Comment (fichtner) [inline security/tor/Makefile]: we don't need this unless you don't want these changes release in 17.7.8

Comment (fabianfrz) [reply security/tor/Makefile]: why not - It does not harm anything

Comment (fichtner) [reply security/tor/Makefile]: because now we are at 1.1 and then we are at 1.3 :P

Comment (fabianfrz) [reply security/tor/Makefile]: no problem, then 1.2 will be a developer preview.

**Pull request merged**